docker-compose.yml

```
@@ -314,6 +316,8 @@ services:
         condition: service_started
       imageprovider:
         condition: service_started
+      flagd:
+        condition: service_started 
```

Please remove the extra space:

```suggestion
        condition: service_started
```

---

thanks for the catch, fixed